share/completions/fab.fish

```diff
@@ -0,0 +1,75 @@
+# tasks 
+for task in (fab -l)
+    set -l _matched (string match -r "^ +([^ ]*) *([^ ]?.*)" $task)
+
+    if test -n "$_matched"
+        set -l _task_name $_matched[2]
+        set -l _task_desc $_matched[3]
+        test -n "$task_desc"; or set -l _task_desc "No description"
+        complete -f -c fab -a "$_task_name" -d "$_task_desc"
```

Comment:
Once this is in a function, you can do `printf "%s\t%s\n" "$_task_name" "$_task_desc" - the part after the tab character (\t) will be used as the description.